spec/features/edit_beer_spec.rb

```diff
@@ -0,0 +1,71 @@
+require "rails_helper"
+
+feature 'edit beer information', %{
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.